### DISCUSSION TITLE
Regression

### DISCUSSION THREAD

**Discussion by wout-konings** [AlexanderFabisch/gmr#48] [Q&A]
Hi, thank you for making this package. However, I am failing to see where I can do a GMM regression with a dependent variable y?

**Comment by AlexanderFabisch** [comment]
Hi @wout-konings 

Here is a simple example: https://github.com/AlexanderFabisch/gmr/blob/master/examples/plot_sklearn_learning_curve.py

Do you have a more specific question?
